packages/web/src/components/Container/README.md

```diff
@@ -0,0 +1,11 @@
+# Container
+
+Container centers your content horizontally and sets its max-width with horizontal paddings.
+
+```html
+<div class="Container">Content</div>
+```
+
+## Theming
+
+Container variables `max-width`, `paddings` and `breakpoints` are mapped to design tokens.
```

Comment:
Good to mention it here, but I'm not sure though—we haven't covered theming in any other (both) existing components. I suggest to leave it off entirely, or to specify what tokens we are talking about. Either way, internal variable names don't have their place here as they are not part of public API.